extensions/interactions/guppy.html

```diff
@@ -1,33 +1,40 @@
-<link rel="stylesheet" href="/third_party/static/guppy-b5055b/build/guppy-default.min.css">
+<link rel="stylesheet" href="/third_party/static/guppy-175999/build/guppy-default.min.css">
 <script>
   // Use the normal default browser eventlistener without passive set to false.
   // TODO(#8833): Use passive listeners when guppy supports it.
   EventTarget.prototype.addEventListener = (
     EventTarget.prototype.addEventListener._original);
 </script>
-<script type="text/javascript" src="/third_party/static/guppy-b5055b/build/guppy.min.js"></script>
-
-<script type="text/javascript">
-  Guppy.init({
-    "path": "/third_party/static/guppy-b5055b/build"
-  });
-</script>
+<script type="text/javascript" src="/third_party/static/guppy-175999/build/guppy.min.js"></script>
 
 <style>
+  /* Some of the style attributes have been marked with '!important' since they
```

Comment:
Add the relevant link to this docstring (make sure that it's to a specific line in a specific commit of guppy) so that it's easier for other devs to investigate in the future if needed.

If you need to, it's fine to break the link across multiple lines.

Reply:
Done.